src/patterns/SaleHeaderBanner/SaleHeaderBanner.tsx

```diff
@@ -67,6 +83,10 @@ const SaleHeaderBanner = forwardRef<HTMLDivElement, SaleHeaderBannerProps>(
       auctionEndTime,
       auctionTitle,
       imageSrcUrl,
+      imageSrcSet,
+      imageSizes,
+      imageLoading,
```

Comment:
should we have defaults here for Loading and Priority?

Reply:
if we want, figured consumer can pass in if they want (the image itself has defaults to `auto` on each one)